pom.xml

```diff
@@ -1896,6 +1896,7 @@ under the License.
 								<exclude>org.apache.flink.streaming.api.environment.LegacyLocalStreamEnvironment</exclude>
 								<exclude>org.apache.flink.streaming.api.functions.sink.RichSinkFunction#invoke(java.lang.Object)</exclude>
 								<exclude>org.apache.flink.streaming.api.functions.sink.SinkFunction</exclude>
+								<exclude>org.apache.flink.streaming.api.operators.StreamOperator</exclude>
```

Comment:
If there come new API files to be modified, can easily exclude those files as this one.